techniques/system/common/1.0/check-rsyslog-version

```diff
@@ -1,31 +0,0 @@
-#!/bin/sh
```

Comment:
it should also be removed from the metadata.xml https://github.com/Normation/rudder-techniques/pull/1617/files#diff-3d9c01bf53ab8104c0bc76244daad2f3R78